Build/Projects/MGCB.definition

```diff
@@ -10,6 +10,8 @@
     <Reference Include="System.Xml" />
     <Reference Include="MonoGame.Framework" />
     <Reference Include="MonoGame.Framework.Content.Pipeline" />
+
+    <Reference Include="MGCB.References" />
```

Comment:
@hach-que - Is there a better way to do this in Protobuild?  We are trying to reference the new DesktopGL version of MonoGame.Framework from the Linux build of the MGCB tool.  Since there is no more Linux build of MonoGame.Framework is there some way to tell Protobuild to always use the DesktopGL version?